cassandra/src/main/java/com/yahoo/ycsb/db/CassandraCQLClient.java

```diff
@@ -381,7 +434,7 @@ public Status scan(String table, String startkey, int recordcount,
 
     } catch (Exception e) {
       e.printStackTrace();
```

Comment:
this should be replaced by passing the exception as a final argument to the logger call below